backend/app/models.py

```diff
@@ -1,15 +1,17 @@
 import enum
+from abc import ABC, abstractmethod
 
+from sqlalchemy import exists
 from sqlalchemy.ext.declarative import declarative_base
 from sqlalchemy.ext.hybrid import hybrid_property
 from werkzeug.security import generate_password_hash, check_password_hash
 
 from init_app import db
-from abc import ABC, abstractmethod
 
 
 class ObjectABC(ABC):
```

Comment:
Object ABC says nothing about AbstractClass, what this abstract class do? Or what kind of classes will use this ABC